### PR TITLE
Add project section and languages.

### PR DESCRIPTION
**Pull request by lornaarmstrong**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -29,55 +28,130 @@ <h3>About Me</h3>
...
+          href="https://www.eusa.ed.ac.uk/our_venues/teviot/" alt="Teviot" target="_blank">
+          <area shape="rec" coords="324,186,451,287" 
+          href="https://en.wikipedia.org/wiki/Calton_Hill" alt="Calton Hill" target="_blank">
+          <area shape="rec" coords="151,229,295,322" 
+          href="https://www.eusa.ed.ac.uk/our_venues/potterrow/" alt="Big Cheese / Potterrow" target="_blank">
+          <area shape="rec" coords="7,175,130,274" 
+          href="https://www.edinburghfirst.co.uk/venues/mcewan-hall/" alt="McEwan Hall" target="_blank">
+        </map>     
       </div>   
+      <!-- This is the links section -->
```
[the remark this answers is not in the document]
Comment (lornaarmstrong) [reply portfolio/src/main/webapp/index.html:65]: Good spot, thank you!

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -29,55 +28,130 @@ <h3>About Me</h3>
...
-        <button class="hyperlinked-button buttonGray">GitHub</button></a>
-      </div>
+      <section class="right-content">
+        <h3>Languages</h3>
+        <p>One of my main hobbies is languages! I hated learning French and German
+          at school, originally, but when I decided to self-teach Greek, aged 14,
+          I changed my mind about languages. No longer did they seem dull, confusing
+          and monotonous to learn! I like to use lots of different techniques, and I
+          especially enjoy reading foreign books, and using Duolingo 
+          <a href="http://duome.eu/larmstrong24" target="_blank">(see my stats!)</a>
```
[the remark this answers is not in the document]
Comment (lornaarmstrong) [reply portfolio/src/main/webapp/index.html:90]: Thanks!

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -29,55 +28,130 @@ <h3>About Me</h3>
...
-            href="https://en.wikipedia.org/wiki/Calton_Hill" alt="Calton Hill" target="_new">
-            <area shape="rec" coords="151,229,295,322" 
-            href="https://www.eusa.ed.ac.uk/our_venues/potterrow/" alt="Big Cheese / Potterrow" target="_new">
-            <area shape="rec" coords="7,175,130,274" 
-            href="https://www.edinburghfirst.co.uk/venues/mcewan-hall/" alt="McEwan Hall" target="_new">
-        </map>      
+          <area shape="rec" coords="23,43,133,119" 
+          href="http://www.pilgrimbar.co.uk/" alt="Pilgrim Bar" target="_blank">
+          <area shape="rec" coords="172,19,311,93" 
+          href="http://www.nms.ac.uk/national-museum-of-scotland/" alt="Museum of Scotland" target="_blank">
```
[the remark this answers is not in the document]
Comment (lornaarmstrong) [reply portfolio/src/main/webapp/index.html:54]: Thanks for the feedback

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -29,55 +28,128 @@ <h3>About Me</h3>
...
+            <h3>Swedish</h3>
+          </div>
+        </div>
+        <img class="extra-border-img" src= "/images/arabic.png" alt="Arabic Flag" width=112px height=75px>
+      </section>
+      <section class="center-content">
+        <div class="buttonText">
+          <a href="https://www.linkedin.com/in/lornafarmstrong" target="_blank">
+          <button class="hyperlinked-button buttonBlue">LinkedIn</button></a>
+          <a href="https://www.https://github.com/lornaarmstrong" target="_blank">
```
Comment (MostafaGabriel) [inline portfolio/src/main/webapp/index.html:148]: please fix this typo before submitting :)

Comment (lornaarmstrong) [reply portfolio/src/main/webapp/index.html]: Will do, thanks!

File: portfolio/pom.xml [hunk 5]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```
Comment (MostafaGabriel) [inline portfolio/pom.xml:37]: You can remove this now :)

**Pull request merged**